sdk/iOS/src/MSQueuePushOperation.m

```diff
@@ -225,6 +225,43 @@ - (void) finishTableOperation:(MSTableOperation *)operation item:(NSDictionary *
                                               internalError:storeError];
                 }
             }
+            else {
+                // Operation was successful, but another operation is pending.
```

Comment:
This would seem to be the place to hook in a local change conflict delegate to resolve any conflicts and allow for preserving server side changes. What do you think about adding something like that here?